pysap/numerics/utils.py

```diff
@@ -15,3 +15,27 @@
 from pysap.plugins.mri.reconstruct.utils import generate_operators
 from pysap.plugins.mri.reconstruct.utils import convert_mask_to_locations
 from pysap.plugins.mri.reconstruct.utils import convert_locations_to_mask
+
+import numpy as np
+
+
+def normalize_samples(samples):
```

Comment:
We would prefer all new functions to be implemented in `plugins` and upon request we can create a shortcut through `numerics` for generic tools that can be used for a variety of applications. So please move this and let us know if you believe it truly generic.

Also please check the formatting of the docstrings. Parameters should be inline with the `"""`.

Reply:
Yes I could put it on a plugins but this would be on the reconstruct plugin because it is supposed to come with the NFFT function, and It will be used everywhere for non-cartesian data, let me know you still want me to put it on a plugin

Reply:
Yes, please move it to the reconstruct plugin. :)